app/forms/password_reset_email_form.rb

```diff
@@ -0,0 +1,38 @@
+class PasswordResetEmailForm
+  include ActiveModel::Model
+  include FormEmailValidator
+
+  attr_accessor :email
+
+  def initialize(email)
+    self.email = email
```

Comment:
are we not using an ivar here because reek doesn't like it? Saw in another PR that Peter also set values with `self.var =` and used `attr_accessor`, which is what I'd usually do with an ivar but he mentioned reek complained. Or is this just a style you prefer / we use in this project? Curious about why!